src/api/db/model.ts

```diff
@@ -17,10 +17,24 @@ export class Build extends Bookshelf.Model<any> {
   get hasTimestamps() { return true; }
   repository() { return this.belongsTo(Repository, 'repositories_id'); }
   jobs() { return this.hasMany(Job, 'builds_id'); }
+  buildRuns() { return this.hasMany(BuildRun, 'builds_id'); }
+}
+
+export class BuildRun extends Bookshelf.Model<any> {
+  get tableName() { return 'build_runs'; }
+  get hasTimestamps() { return true; }
+  buildRun() { return this.belongsTo(BuildRun, 'build_id'); }
```

Comment:
`BuildRun` cannot belong to itself. I believe you wanted `build() { return this.belongsTo(Build, 'build_id'); }`.
and then in `Build`:
```ts
export class Build extends Bookshelf.Model<any> {
  get tableName() { return 'builds'; }
  get hasTimestamps() { return true; }
  repository() { return this.belongsTo(Repository, 'repositories_id'); }
  jobs() { return this.hasMany(Job, 'builds_id'); }
  runs() { return this.hasMany(BuildRun, 'builds_id'); }
}
```
note `runs()` instead of `buildRuns()` to be consistent.

Same thing apply to Job please.